debezium-storage/debezium-storage-jdbc/src/main/java/io/debezium/storage/jdbc/history/JdbcSchemaHistory.java

```diff
@@ -126,7 +126,7 @@ protected void storeRecord(HistoryRecord record) throws SchemaHistoryException {
                     conn.rollback();
                 }
                 catch (SQLException ex) {
-                    // ignore
+                    LOG.error("Exception during storeRecord", ex);
```

Comment:
We throw the exception on the next line, no need to log anything here, it just duplicates the exception in the log.

Reply:
reverted.